packit_service/worker/github_handlers.py

```diff
@@ -75,6 +75,29 @@ class AbstractGithubJobHandler(JobHandler):
     pass
 
 
+def _get_package_config_from_repo(
+    project: GitProject, reference: str, pr_id: int = None
+):
+    try:
+        package_config: PackageConfig = get_package_config_from_repo(project, reference)
+        if not package_config:
+            raise PackitConfigException(
+                f"No config file found in {project.full_repo_name}"
+            )
+    except PackitConfigException as ex:
+        if pr_id:
+            project.pr_comment(pr_id, ex)
+        else:
+            # TODO: filter when https://github.com/packit-service/ogr/issues/308 fixed
+            issues = project.get_issue_list()
+            if "Invalid packit config" not in [x.title for x in issues]:
+                # TODO: store in DB
+                i = project.create_issue("[packit] Invalid config", ex)
```

Comment:
We can enhance the format later, but some ideas:

- formating -- is it "safe" to print the exception directly? (Maybe yes.) Or use a ``` block.
- links to the documentation
- contact with our team
- info about the issue workflow (e.g. "We will try not to do anything until you close the issue...")